src/fsharp/vs/service.fs

```diff
@@ -1473,12 +1473,10 @@ type TypeCheckInfo
                // custom builders, custom operations get colored as keywords
                | CNR(_, (Item.CustomBuilder _ | Item.CustomOperation _), ItemOccurence.Use, _, _, _, m) -> 
                    yield (m, FSharpTokenColorKind.Keyword) 
-#if COLORIZE_TYPES
```

Comment:
ATM, we still need intellisense for type colorization to work. I've chosen to start this in a separate PR, because the current one is getting huge already. I'm going to rewrite/refactor the type colorization code in Colorize.fs (is this the one you meant?) and add many more tests to check for this specifically.